.github/ISSUE_TEMPLATE/3.rfc.yml

```diff
@@ -0,0 +1,47 @@
+name: RFC 💬
+description: Request for comments for your proposal.
+title: "[RFC] "
+labels: ['RFC']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the RFC in the title above ⬆️.
+
+        Thanks for contributing by creating an RFC! ❤️
+  - type: textarea
+    attributes:
+      label: What's the problem? 🤔
+      description: A short paragraph or bullet list that quickly explains what you're trying to do, what value do we expect to get or any other relevant information to understand motivation behind this RFC
+
+  - type: textarea
+    attributes:
+      label: What are the requirements? ❓
```

Comment:
We could consider this as part of the problem statement that is above. At least, it's how it's currently done inside https://www.notion.so/mui-org/Shaping-33647d13ed534b838bde5bc8182eba18, I think to keep the template as open ended as possible.

Reply:
I'd need to pull in the exact conversation but when we discussed this it was more about listing requirements in it's explicit section. For shaping items it might be less important, but since it's code related your might have certain constraints that you wanna list more often then not.
We could embed this into "problem statement" but I find having it in separate section is easier to find just by scanning 🤔

Reply:
> but since it's code related your might have certain constraints that you wanna list more often then not.

I think it's very similar for non-code-related topics, e.g. https://www.notion.so/mui-org/Scale-collaborative-inbox-in-Google-Groups-f6f3d7a8a6bd4c24be1fc8b13a8f33f6#7d75b82e0525464c999d5bde5de6a6c2.

Reply:
It is indeed similar to what you show - but how would we formulate this explicitly in the problem statement? Do we expect that everyone will implicitly understand to list the requirements?
Should it simply just be "Requirements" subsection under problem statement?

Reply:
> Should it simply just be "Requirements" subsection under problem statement?

It was what I was wondering about, maybe an opportunity to keep the RFC layout simpler 🤷‍♂️, e.g. https://github.com/reactjs/rfcs/blob/main/0000-template.md has no notions of requirements. But no real preferences.